randovania/data/json_data/prime3.txt

```diff
@@ -1530,13 +1791,20 @@ Asset id: 1872319291341539278
 > Dock to Jungle Generator; Heals? False
   * Normal Door to Jungle Generator/Dock to Generator Hall South
   > Dock to South Jungle Court
+      After Generator Hall South Bridge
+  > Event - Destroy Bridge
       Trivial
 
 > Dock to South Jungle Court; Heals? False
   * Normal Door to South Jungle Court/Dock to Generator Hall South
   > Dock to Jungle Generator
       Trivial
 
+> Event - Destroy Bridge; Heals? False
+  * Event Generator Hall South Bridge
+  > Dock to Jungle Generator
+      Trivial
+
 ----------------
 South Jungle Hall
```

Comment:
I can't comment where I want to because it wasn't edited, but for Save Station B, add a node called "Save Station", enable the "Heals?" checkbox, and link it to the door as "Trivial".